code/mission/missionparse.cpp

```diff
@@ -5686,6 +5695,11 @@ void parse_asteroid_fields(mission *pm)
 		} else {
 			Asteroid_field.has_inner_bound = 0;
 		}
+
+		if (optional_string("$Asteroid Targets:")) {
+			stuff_string_list(Asteroid_target_ships);
+			Default_asteroid_throwing_behavior = false;
+		}
```

Comment:
Add the equivalent saving code to missionsave.cpp (both FRED and qtFRED versions) so that if you load and save a mission, the asteroid target data won't be wiped.  This will allow missions to use the feature in a stable way, even without UI support.

Reply:
I've haven't done FRED mission saving stuff before, and I can't currently compile either so please make sure I did this correctly.